twoopstracker/authentication/serializers.py

```diff
@@ -15,6 +15,9 @@ def validate_email(self, value):
 
 
 class CustomRegisterSerializer(RegisterSerializer):
+    first_name = serializers.CharField(allow_blank=True, max_length=35)
```

Comment:
1. Do we want names to be blank on registration?
2. Length seem kind of short… Does it match what is defined in User model?

Reply:
@kilemensi 
1. On Twoops Frontend, both first_name & last_name are required. These are generally optional fields, so I thought on the API they can remain optional.
2. I will update it, I think in User model its 150...just checked.